aptos-move/move-examples/fungible_asset/sources/coin.move

```diff
@@ -0,0 +1,48 @@
+/// This is an example showing how to create a fungible asset and how to use it.
+module fungible_asset::coin {
```

Comment:
does this exist purely for demos and testing? I think this can be incorporated as an integration test

Reply:
showing ppl how to use it as a demo